pkg/openthread/contrib/netdev/openthread_netdev.c

```diff
@@ -83,12 +88,8 @@ static void *_openthread_event_loop(void *arg) {
     netdev_t *dev;
     msg_t msg, reply;
 
+#if defined(MODULE_LIBOPENTHREAD_CLI_FTD) || defined(MODULE_LIBOPENTHREAD_CLI_MTD)
```

Comment:
I don't understand how and where modules LIBOPENTHREAD_CLI_FTD and LIBOPENTHREAD_CLI_MTD are defined ?